eplaunch/interface/frame.py

```diff
@@ -1236,7 +1236,7 @@ def handle_specific_documentation_menu(self, event):
     def handle_tb_explorer(self, event):
         current_platform = Platform.get_current_platform()
         if current_platform == Platform.WINDOWS:  # pragma: no cover
-            os.system('start "{}"'.format(self.selected_directory))
+            os.startfile('{}'.format(self.selected_directory))
```

Comment:
This seems fine, though if `self.selected_directory` is just a string, it could even be simpler:
```
os.startfile(self.selected_directory)
```